dss-server/src/test/java/io/github/ztkmkoo/dss/server/rest/DssRestServerTest.java

```diff
@@ -95,8 +101,30 @@ public void testSsl() throws Exception {
         assertTrue(dssRestServer.isShutdown());
     }
 
+    @Test
+    public void testJsonPostRequest() throws IOException, InterruptedException {
+        final DssRestServer dssRestServer = new DssRestServer("127.0.0.1", 8181);
+
+        startOnDssJsonRestServer(dssRestServer);
+        await().until(dssRestServer::isActivated);
```

Comment:
Hi, 
If there are some bugs, and dssRestServer do not activated. I'm afraid the test will be pending.

Why not let await have timeout or let junit test have timeout property?

Reply:
Thank you for your review.

I'll try it.